packages/theming/theme-types/src/Color.types.ts

```diff
@@ -846,83 +846,236 @@ export interface AliasColorTokens {
 
   /// Red
 
-  /** @platform macOS, win32, windows */
+  // TODO #2765: remove from macOS
+  /** @platform macOS */
   redBackground1?: ColorValue;
```

Comment:
Removing this would be a breaking change so I would make sure we mark it as deprecated even after macOS updates to the new color.

Also we might reuse these anyway because I believe there are shared color alias tokens

Reply:
Sure will mark them as deprecated when I remove from macOS. What shared color alias tokens are you referring to?

Reply:
https://react.fluentui.dev/?path=/docs/theme-colors--page

towards the end of this page a bunch of color tokens are defined. They're not necessarily using the names we would use, but they are defined in the alias token space in the fluent figma.